gunicorn/config.py

```diff
@@ -346,6 +346,15 @@ def validate_pos_int(val):
     return val
 
 
+def validate_float(val):
+    if not val:
+        return 0.0
```

Comment:
I think this is OK b/c I specify a default in `SpawnWorkerSleepTime`, right?  Adjusted the default value to more closely match the existing behavior.